src/main/java/org/opensearch/security/ssl/DefaultSecurityKeyStore.java

```diff
@@ -879,10 +887,24 @@ private SslContext buildSSLServerContext(final PrivateKey _key, final X509Certif
                                              final X509Certificate[] _trustedCerts, final Iterable<String> ciphers, final SslProvider sslProvider,
                                              final ClientAuth authMode) throws SSLException {
 
-        final SslContextBuilder _sslContextBuilder = SslContextBuilder.forServer(_key, _cert).ciphers(ciphers)
-            .applicationProtocolConfig(ApplicationProtocolConfig.DISABLED)
+        final SslContextBuilder _sslContextBuilder = SslContextBuilder.forServer(_key, _cert)
+            .ciphers(Stream
+                .concat(
+                    Http2SecurityUtil.CIPHERS.stream(),
+                    StreamSupport.stream(ciphers.spliterator(), false))
+                .collect(Collectors.toSet()), SupportedCipherSuiteFilter.INSTANCE)
             .clientAuth(Objects.requireNonNull(authMode)) // https://github.com/netty/netty/issues/4722
-            .sessionCacheSize(0).sessionTimeout(0).sslProvider(sslProvider);
+            .sessionCacheSize(0).sessionTimeout(0).sslProvider(sslProvider)
+            .applicationProtocolConfig(
+                new ApplicationProtocolConfig(
```

Comment:
Looks like this default configuration is repeated down below. Should we consider making this a static constant?

Reply:
May be better as a function if we consider future possibility to configure `SelectorFailureBehavior` and `SelectedListenerFailureBehavior`?

Reply:
Sounds reasonable.

Reply:
Done, thanks @cwperks !